### DISCUSSION TITLE
How do you count team members in the new Business plan?

### DISCUSSION THREAD

**Discussion by jobee** [plausible/analytics#3529] [Anything Else]
I'd really like to migrate my legacy plan (with unlimited sites & team members) to the new Business plan for all my sites. But I'm wondering how you count the limit of 10 team members.

Is it unique email addresses across all sites or max. members per site?

**Comment by metmarkosaric** [comment]
hi @jobee! it's unique emails so basically having the same person added to 10 different sites takes only 1 team member

**Reply by jobee** [reply]
Thanks for your quick reply.
Guess we have to talk about the Custom plan then :-/

Is there a difference between admin & viewer roles?
Is there a price tag for additional user seats? We really don't need any other custom/enterprise feature.

**Reply by metmarkosaric** [reply]
Np!

No difference between admin or viewer roles. One unique team member regardless of the role takes one seat.

There is a price tag for additional seats. You can [contact us](https://plausible.io/contact) with your needs and we'll get back to you with details. 

Note that as you say that you're on the grandfathered plan, you will keep access to your original limit of team members even after you upgrade to a business plan.

**Reply by jobee** [reply]
> Note that as you say that you're on the grandfathered plan, you will keep access to your original limit of team members even after you upgrade to a business plan.

Problem solved! :-) Thought Business would be actually a downgrade in sites & team members for me.
Upgrading soon.
